cell/processes/death.py

```diff
@@ -351,4 +351,5 @@ def plot_death_freeze_state_test():
 
 
 if __name__ == '__main__':
+    test_death_freeze_state()
```

Comment:
This looks like leftover debugging code?

Reply:
It is indeed.